src/components/Footer/index.tsx

```diff
@@ -26,23 +29,30 @@ function Component() {
             <p>
                 Copyright &copy; {copyrightYear} {SITE_NAME}
             </p>
-            <div className="d-flex">
+            <div className="d-flex align-items-center">
+                <TwitterShareButton
+                    className="text-primary fs-5"
+                    title="Save yourself hundreds of thousands of dollars and years wasted on a traditional MBA, and instead learn the principles that billionaires have mastered. Check out the secrets of their success at"
```

Comment:
Please move this string to `constants/social`

Reply:
done